plugins/nodes/src/js/components/NodesGridView.js

```diff
@@ -86,8 +86,7 @@ var NodesGridView = React.createClass({
 
     // Add 'Others' node to the list
     if (activeServiceIds.length > MAX_SERVICES_TO_SHOW) {
-      var classNameOther =
-        `dot service-color-${OTHER_SERVICES_COLOR}`;
+      var classNameOther = `dot service-color-${OTHER_SERVICES_COLOR}`;
```

Comment:
shall we do `const` as we're touching the line anyway?

Reply:
I'd not touch it, it made it to master some how and now our tooling changes it back in every PR, changing it in one would result in a ton of conflicts… :(

Reply:
maybe we should improve our prettier tooling, it seems to forget some files.

Reply:
I'd love to trace the leak and see how we can prevent both `var` and non-prettified changes on master.

Reply:
I think eslint is missing something, that's why I would love to move it into jest as a runner to have a central place where these file paths are stored

Reply:
But that's a completely different topic

Reply:
@nLight @DanielMSchmidt sure, can one of you pick this up? 🙇

Reply:
on it

Reply:
https://github.com/dcos/dcos-ui/pull/3160